### PR TITLE
chore: followup after #104

### PR DESCRIPTION
**Pull request by thymikee**

### CODE REVIEW

File: packages/cli/src/eject/eject.js [hunk 1]
```diff
@@ -91,7 +92,8 @@ function eject() {
       path.resolve(
         'node_modules',
         'react-native',
-        'local-cli',
+        '@react-native-community',
+        'cli',
```
Comment (thymikee) [inline packages/cli/src/eject/eject.js]: looks like we don't have a correct test for this scenario

Comment (grabbou) [reply packages/cli/src/eject/eject.js]: True. I don't even know when this command is being called.

File: packages/cli/src/generator/copyProjectTemplateAndReplace.js [hunk 2]
```diff
@@ -17,7 +17,7 @@ const walk = require('../util/walk');
  * Util for creating a new React Native project.
  * Copy the project from a template and use the correct project name in
  * all files.
- * @param srcPath e.g. '/Users/martin/AwesomeApp/node_modules/react-native/local-cli/templates/HelloWorld'
+ * @param srcPath e.g. '/Users/martin/AwesomeApp/node_modules/react-native/templates/HelloWorld'
```
Comment (grabbou) [inline packages/cli/src/generator/copyProjectTemplateAndReplace.js]: chore: `react-native/template` instead of `templates/HelloWorld`.

Comment (thymikee) [reply packages/cli/src/generator/copyProjectTemplateAndReplace.js]: fixd

**Pull request merged**